calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx

```diff
@@ -0,0 +1,22 @@
+xx May 2023
+
+### What's new {#whats-new-3.17.0}
+
+#### New policy recommendations engine for namespace isolation
+{{prodname}} has added a new policy recommendations engine that automatically generates policies for namespace isolation within your cluster. [Policy recommendations](network-policy/recommendations/policy-recommendations.mdx).
+
+#### Support for DNS rules in clusters using NodeLocal DNSCache
+{{prodname}} has added support for DNS rules in clusters using NodeLocal DNSCache. Also related, there is new documentation on using Calico policy to secure DNS traffic within the cluster with NodeLocal DNSCache enabled. [Use NodeLocal DNSCache in your cluster](networking/configuring/node-local-dns-cache.mdx).
+
+#### Improved UI for configuring Workload-based Web Application Firewall (WAF)
+{{prodname}} includes updates to the UI that allows you to select which services are enabled for the Workload-based Web Application Firewall. [Web application firewall](threat/web-application-firewall.mdx).
+
+#### Wireguard support for AKS and EKS with Calico CNI
+{{prodname} now offers official support for Wireguard when using Microsoft AKS or Amazon EKS with Calico CNI. This mode of deployment offers performance benefits and a more efficient routing table compared to using cloud provider CNIs.  [Encrypt data in transit](compliance/encrypt-cluster-pod-traffic.mdx).
+
+#### Helm support for BYO Prometheus Operator
+{{prodname} now supports an existing Prometheus Operator when installing with Helm. [Helm install](getting-started/install-on-clusters/kubernetes/helm.mdx).
```

Comment:
In the dm with @tigeraphil and @bmckercher123 we mentioned this as a possible note:

```
Added support for BYO Prometheus Operator when installing with Helm. To do this, either follow existing instructions for customizing the helm chart or simply pass the following flags to the installation command:
$ helm install calico-enterprise tigera-operator-vx.y.z-0.tgz ... \
    --set tags.tigera-prometheus-operator=false \ 
    --set tigera-prometheus-operator.enabled=false
```

I am mainly referring to the second sentence, would that help the user if they decide to use this?

Reply:
The downside is potentially having to update the command two places, but I think it does help users see how easy it is. Suggest:

{{prodname} now supports an existing Prometheus Operator when installing with Helm. Just pass the false flags to the installation command, for example:
```
$ helm install calico-enterprise tigera-operator-vx.y.z-0.tgz ... \
    --set tags.tigera-prometheus-operator=false \ 
    --set tigera-prometheus-operator.enabled=false
```
Or customize the Helm chart. [Helm install](getting-started/install-on-clusters/kubernetes/helm.mdx).